src/App.jsx

```diff
@@ -18,19 +19,38 @@ function App() {
       "https://vignette.wikia.nocookie.net/mrmen/images/4/4f/MR_JELLY_4A.jpg/revision/latest?cb=20180104121141",
   });
 
+  const [error, setError] = useState(null);
+  const handleError = (errorMessage) => {
```

Comment:
You created a state and function to update it here for the error and passed it down to each of the components but none of them are using it? In this case we can delete it :)